taigit/src/libraries/Taiga.ts

```diff
@@ -78,4 +78,23 @@ task_history(taskId : number) : Promise<Object> {
     }
 
     return output;
+}
+
+/**
+ * @summary Get the all the user story name
+ * @param slug project/slug name
+ * @returns array of user-stories name
+ * {
+ *  userStory1, userStory2,...
+ * }
+ */
+export async function
+getUserStoryTitles(slug: string): Promise<Object> {
+    let projectId = (await axios.get(`https://api.taiga.io/api/v1/projects/by_slug?slug=${slug}`)).data.id;
```

Comment:
based on frontends reply the project id should be passed in instead of the slug

Reply:
Done!